public/src/actions/calls.jsx

```diff
@@ -107,16 +107,21 @@ export function releaseCall(userId, campaignId, callId, currentCallStatus, next
   .catch(err => err);
 }
 
-export function updateCallAttempt(userId, campaignId, callId, outcome, notes = null) {
-  const params = { outcome, notes };
+export function updateCallAttempt(userId, campaignId, callId, status, outcome, notes) {
+  const params = { outcome, notes, status };
   return dispatch => axios.put(`/users/${userId}/campaigns/${campaignId}/calls/${callId}`,
     params,
     {
       headers: { Authorization: ` JWT ${localStorage.getItem('auth_token')}` }
     }
   )
-  .then(() => {
-    dispatch(assignToCall(userId, campaignId));
+  .then((currentCall) => {
+    const { call: currentCallObj } = currentCall.data;
+    const { status: currentCallStatus } = currentCallObj;
+    if (currentCallStatus === 'ATTEMPTED') {
```

Comment:
Hmm maybe let's leave this out? Because it's a hidden side effect? I think a calling method should explicitly call assignToCall (such as in the submit call action).

Reply:
Also, can we update the test for this action?

Reply:
✔️